server.js

```diff
@@ -1,33 +1,85 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import netflixData from './data/netflix-titles.json';
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+console.log(netflixData.length);
+
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send(
+    'paths: movies, movies/movie, genres/genre, actors/actor, directors/director, year/year, countries/country'
+  );
+});
+
+app.get('/movies', (req, res) => {
+  res.json(netflixData);
+});
+
+// route for getting data on a certain movie
+app.get('/movies/:movie', (req, res) => {
+  const movie = req.params.movie;
+  const chosenMovie = netflixData.filter(
+    (item) => item.title.toString().toLowerCase() === movie.toLowerCase()
+  );
+  res.json(chosenMovie);
+});
+
+// route for getting movies from a certain category
+app.get('/genres/:genre', (req, res) => {
+  const genre = req.params.genre;
+  const movieGenre = netflixData.filter((item) =>
+    item.listed_in.toLowerCase().includes(genre.toLowerCase())
+  );
+  res.json(movieGenre);
+});
```

Comment:
It would be nice if this also was a filter on the `/movies` endpoint, e.g. `/movies?genre=drama`